crates/common/fork_choice/src/store.rs

```diff
@@ -145,7 +164,12 @@ impl Store {
 
     /// Retrieve a filtered block tree from ``store``, only returning branches
     /// whose leaf state's justified/finalized info agrees with that in ``store``.
-    pub fn get_filtered_block_tree(&self) -> anyhow::Result<HashMap<B256, BeaconBlock>> {
+    ///
+    /// NOTE: ``blocks`` must contain justified/finalized epoch information of its node, so tuple of
+    /// `(BeaconBlock, justified_epoch, finalized_epoch)` should be the value of the map.
+    pub fn get_filtered_block_tree(
+        &self,
+    ) -> anyhow::Result<HashMap<B256, (BeaconBlock, u64, u64)>> {
```

Comment:
could you just make a struct for (BeaconBlock, u64, u64)? as it is a little non-intuitive